helpers.cpp

```diff
@@ -323,7 +330,12 @@ void mach::SocketConn::input() {
     if (val == "op6") {
       poolptr->stop();
       std::cout<<"past stop" <<std::endl;
-      std::terminate();
+      std::terminate(); // ?
```

Comment:
we can just kill this line it's unnecessary

Reply:
oh yeah I was gonna ask how the abort sequence runs if we call std::terminate(), but if its unnecessary then no problemo

Reply:
oh yeah I was gonna ask how the abort sequence runs if we call std::terminate(), but if its unnecessary then no problemo